client/matchmaking/matchmaking-disabled-card.tsx

```diff
@@ -152,6 +130,35 @@ export function ConnectedMatchmakingDisabledCard({
       ) : (
         <Headline6>Soon™</Headline6>
       )}
-    </Contents>
+    </DisabledCard>
+  )
+}
+
+export interface PartyDisabledCardProps extends DisabledCardProps {
+  isPartyLeader: boolean
+  partySize: number
+}
+
+export function PartyDisabledCard({
```

Comment:
imo these components should either all be connected or none of them should be connected, it's confusing that this one needs all these props and the other one pulls the info out of the store itself